SilasKenneth/fast-food-fast-ui
# [Finishes #160136042] Show user order history

#### What this pull request does
Creates a page where a user can view his/her order history
##### Description of the task to be completed
 Create a page where a logged in user can see his/her order history
    -  Add links for the logged in user
    - Add links for the user to choose other options
##### Relevant pivotal tracker story
[#160136042](https://www.pivotaltracker.com/n/projects/2194200/stories/160136042)
##### How to test
  - Clone the repo ``git clone https://github.com/SilasKenneth/fast-food-fast-ui.git``
  -  Go to the fast-food-fast-ui directory and head to UI folder
  - Open the userorders.html file in your browser and that should work

#### Screenshot
![screencapture-localhost-3000-userorders-html-2018-08-31-23_12_34](https://user-images.githubusercontent.com/11555354/44935640-3e70df00-ad7a-11e8-896a-ce66d4c7dbda.png)
